src/CalendarStrip.js

```diff
@@ -145,6 +145,20 @@ class CalendarStrip extends Component {
   componentDidMount() {
     // Animate showing of CalendarDay elements
     this.animate();
+
+    selectedDate = {
+      selectedDate: this.setLocale(moment(this.props.selectedDate))
+    };
+    startingDate = {
+      startingDate: this.updateWeekStart(selectedDate.selectedDate)
+    };
+    weekData = this.updateWeekData(
+      startingDate.startingDate,
+      selectedDate.selectedDate,
+      this.props
+    );
+
+    this.setState({...weekData});
```

Comment:
For the case of selectedDate prop not provided, this should not run because moment(undefined) creates today's date.  The entire block should be wrapped in `if (this.props.selectedDate)`